src/client/job.rs

```diff
@@ -66,6 +68,40 @@ fn status_cell(status: Status) -> CellStruct {
     }
 }
 
+/// Draws a colored progress bar that depicts counts of tasks with individual states
+fn job_progress_bar(info: &JobInfo) -> String {
+    let mut buffer = String::from("[");
+
+    let width: usize = 40;
+    let parts = vec![
```

Comment:
* Do you need to construct a Vec, isnt just an array sufficient?
* Color settings should respect gsettings.color_policy

Reply:
- Array will be possible to use here from Rust 1.53, which will be released.. in about an hour :)
- Color settings are set [here](https://github.com/It4innovations/hyperqueue/pull/60/files#diff-68808a0be41327ce142404d37bbf8f747794caa49909229052f8fbf9a284df09R284).